frontend/src/service/Url.service.ts

```diff
@@ -190,15 +196,14 @@ export class UrlService {
   private gqlCreateURLVariable(
     captchaResponse: string,
     link: Url,
-    alias: string | null,
     isPublic: boolean = false
   ) {
     return {
       captchaResponse: captchaResponse,
       authToken: this.authService.getAuthToken(),
       urlInput: {
         originalURL: link.originalUrl,
-        customAlias: alias
+        customAlias: link.alias === '' ? null : link.alias!
```

Comment:
Let's move this logic up and assign it a dedicated variable. Ideally avoid `ternary operator` when possible.

Reply:
TODO
- [ ] Assign `link.alias === '' ? null : link.alias!` to its own variable to avoid use of ternaries